stack-ghc-9.4.5.yaml

```diff
@@ -1,7 +1,7 @@
 # This is an experimental project-level configuration, to see if Stack can be
 # built with GHC 9.4.5.
 
-resolver: nightly-2023-06-18 # GHC 9.4.5
+resolver: lts-21.1
```

Comment:
I sneaked this one into the series as I use GHC 9.4 at work and wanted to avoid the download for 9.2 as  I was on a low througput internet connection.